labelbox/schema/model_run.py

```diff
@@ -262,6 +262,39 @@ def update_status(self,
             },
             experimental=True)
 
+    @experimental
+    def update_config(self, config: Dict[str, Any]):
+        data: Dict[str, Any] = {'config': config}
+        res = self.client.execute(
+            """mutation updateModelRunConfigPyApi($modelRunId: ID!, $data: UpdateModelRunConfigInput!){
+                updateModelRunConfig(modelRun: {id : $modelRunId}, data: $data){trainingMetadata}
+            }
+        """, {
+                'modelRunId': self.uid,
+                'data': data
+            },
+            experimental=True)
+        return res["updateModelRunConfig"]
+
+    @experimental
+    def reset_config(self):
+        res = self.client.execute(
+            """mutation resetModelRunConfigPyApi($modelRunId: ID!){
+                resetModelRunConfig(modelRun: {id : $modelRunId}){trainingMetadata}
+            }
+        """, {'modelRunId': self.uid},
+            experimental=True)
+        return res["resetModelRunConfig"]
+
+    @experimental
+    def fetch_config(self):
```

Comment:
Suggestion: `get_config` would follow the general naming convention of SDK methods better, unless the PM has already signed off on `fetch_config`